gapis/memory/alignof_sizeof.go

```diff
@@ -29,7 +29,7 @@ func AlignOf(t reflect.Type, m *device.MemoryLayout) uint64 {
 		return uint64(m.GetPointer().GetAlignment())
 	case t == tyInt, t == tyUint:
 		return uint64(m.GetInteger().GetAlignment())
-	case t == tySize:
+	case t.Implements(reflect.TypeOf((*SizeTy)(nil)).Elem()):
```

Comment:
In 89556d0, the tySize is replaced with `reflect.TypeOf((*SizeTy)(nil)).Elem()`, and all the `==` type comparisons are changed to something like `t.Implements(CharTy):`.

Any new types defined with `type xxx ϟmem.Char` will needs to implements `IsChar()`. However, I have not found any of such types in the code.